packages/turbo/src/Session.tsx

```diff
@@ -1,100 +0,0 @@
-import { getNativeModule, registerMessageEventListener } from './common';
-import React, { RefObject } from 'react';
-import type { EmitterSubscription, NativeSyntheticEvent } from 'react-native';
-import { SessionContext } from './SessionContext';
-import type {
-  SessionModule,
-  SessionMessageCallback,
-  OnErrorCallback,
-} from 'packages/turbo/src/types';
-
-const RNSessionModule = getNativeModule<SessionModule>('RNSessionModule');
-
-interface Message {
-  message: object;
-}
-
-export interface Props {
-  onMessage?: SessionMessageCallback;
-  onVisitError?: OnErrorCallback;
-}
-
-interface State {
-  sessionHandle?: string | null;
-}
-
-export default class Session extends React.Component<Props, State> {
```

Comment:
as discussed - for better developer experience I would keep this file (until v1.0) that would put `console.error` + throw an error with link to this PR description that would explain how to change existing codebases (for those few individuals who uses our lib)